scripts/DB_Migration.sh

```diff
@@ -4,7 +4,7 @@ docker cp $(pwd)/MISC mongo:/
 docker exec -i mongo mongoimport \
 	--db VercipeDB \
     --drop --collection Recipes \
-	--file /MISC/db_backup/recipes.json \
+	--file /MISC/db_backup/Recipes.json \
```

Comment:
why? the file in master branch is not capitalized

Reply:
it is

Reply:
hummm... wired